ngraph/frontend/onnx_import/include/onnx_import/utils/parser.hpp

```diff
@@ -0,0 +1,45 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
```

Comment:
Just friendly reminder about the new year, same for other files:)
```suggestion
// Copyright 2017-2021 Intel Corporation
```

Reply:
I've created a PR with batch-update of copyright information https://github.com/openvinotoolkit/openvino/pull/3832